app/db.js

```diff
@@ -8,29 +8,30 @@ const dbUrl = nconf.get('db');
 mongoose.Promise = Promise;
 
 const connect = function () {
-  const options = {
+  /**
+   const options = {
     server: {
       socketOptions: {keepAlive: 1},
       auto_reconnect: true
     }
-  };
-  /** @todo figure out valid configurations
-    const options = {
-      useMongoClient: true,
-      keepAlive: 120,
-      autoReconnect: true,
-      logger: logger,
-      loggerLevel: 'warning'
   }; */
+  const options = {
+    useMongoClient: true,
+    keepAlive: 120,
+    autoReconnect: true,
+    logger: logger,
```

Comment:
this require little different logger APi's: http://mongodb.github.io/node-mongodb-native/2.2/api/Logger.html . Options: support those additional api's for ClusterLogger or remove logger options here.